
---
moveit_msgs/MoveItErrorCodes error_code
---
#feedback
int32 status